### PR TITLE
G Suite: Show badge for admin users on Email page

### PR DESCRIPTION
**Pull request by stephanethomas**
This pull request updates the [`Email` page](https://wordpress.com/email) to show a badge in front of each G Suite user that has super admin privileges:

screenshot

Those changes were extracted from https://github.com/Automattic/wp-calypso/pull/46011 which was getting too big. The badge itself is just a nice-to-have, the main improvement here is that we are now able to determine whether a G Suite user is an admin or not. This information has been returned by the API for several weeks now but had not been surfaced in Calypso until now.

Note this pull request also fixes a warning because of props no longer required, and renames a variable in order to better convey its purpose.

#### Testing instructions

> You can change the role of a G Suite user from the the `Users` section of your [Google Admin](https://admin.google.com/). For a given user, locate and click the `Admin roles and privileges` section, then click the `Roles` section and finally unassign the `Super Admin` role. Please note that we cache information from fifteen minutes to one hour on our side, so you will either have to wait that time to see your changes, or to clear the cache manually (ping me if you need help with this).

1. Run `git checkout add/gsuite-admin-badges` and start your server, or open a [live branch](https://calypso.live/?branch=add/gsuite-admin-badges)
2. Log into a WordPress.com account with a site that has G Suite
3. Open the [`Email` page](http://calypso.localhost:3000/email) for that site
4. Assert that you see a badge in front of each admin user

### CODE REVIEW

File: client/state/gsuite-users/schema.js [hunk 1]
```diff
@@ -5,34 +5,36 @@ export const usersSchema = {
...
 					email: { type: 'string' },
 					firstname: { type: 'string' },
-					lastname: { type: 'string' },
 					fullname: { type: 'string' },
+					is_admin: { type: 'boolean' },
+					lastname: { type: 'string' },
+					mailbox: { type: 'string' },
 					site_id: { type: 'integer' },
 					suspended: { type: 'boolean' },
-					agreed_to_terms: { type: 'boolean' },
```
Comment (stephanethomas) [inline client/state/gsuite-users/schema.js:27]: I've only added the new `is_admin` property in this schema, the rest of the changes is just ordering properties alphabetically so they are easier to read.

File: client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx [hunk 2]
```diff
@@ -124,9 +124,7 @@ class PendingGSuiteTosNotice extends React.PureComponent {
 		const button = (
 			<PendingGSuiteTosNoticeAction
 				domainName={ domainName }
-				isMultipleDomains={ false }
 				section={ this.props.section }
-				severity={ null }
```
Comment (stephanethomas) [inline client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx:129]: This was causing the following warning:

```javascript
Warning: Failed prop type: The prop `severity` is marked as required in `PendingGSuiteTosNoticeAction`, but its value is `null`.
  in PendingGSuiteTosNoticeAction (created by PendingGSuiteTosNotice)
  in PendingGSuiteTosNotice (created by Context.Consumer)
  in WithLocalizedMoment(PendingGSuiteTosNotice) (created by Localized(WithLocalizedMoment(PendingGSuiteTosNotice)))
  in Localized(WithLocalizedMoment(PendingGSuiteTosNotice)) (created by ConnectFunction)
  in ConnectFunction (created by RegisteredDomainType)
  in div (created by Card)
  in Card (created by DomainStatus)
  in DomainStatus (created by RegisteredDomainType)
  in div (created by RegisteredDomainType)
  in RegisteredDomainType (created by Localized(RegisteredDomainType))
  in Localized(RegisteredDomainType) (created by Context.Consumer)
  in WithLocalizedMoment(Localized(RegisteredDomainType)) (created by ConnectFunction)
  in ConnectFunction (created by Edit)
  in main (created by Main)
  in Main (created by Edit)
  in Edit (created by Localized(Edit))
  in Localized(Edit) (created by ConnectFunction)
  in ConnectFunction (created by StoreConnection)
  in StoreConnection (created by DomainManagementData)
  in div (created by DomainManagementData)
  in DomainManagementData (created by ConnectFunction)
  in ConnectFunction
  in div (created by Layout)
  in div (created by Layout)
  in div (created by Layout)
  in Layout (created by ConnectFunction)
  in ConnectFunction (created by WithCurrentRoute(Connect(Layout)))
  in WithCurrentRoute(Connect(Layout)) (created by ProviderWrappedLayout)
  in MomentProvider (created by ConnectFunction)
  in ConnectFunction (created by ProviderWrappedLayout)
  in Provider (created by ProviderWrappedLayout)
  in RouteProvider (created by ProviderWrappedLayout)
  in I18nProvider (created by CalypsoI18nProvider)
  in CalypsoI18nProvider (created by ProviderWrappedLayout)
  in ProviderWrappedLayout
```

File: client/state/gsuite-users/schema.js (hunk 1 above)
Comment (olaseni) [reply client/state/gsuite-users/schema.js:27]: Thanks for surfacing this fact. It made things easier.

**Review by olaseni — APPROVED** [review]
Looks good and tests well.

**Pull request merged**